pycolleff/pycolleff/colleff.py

```diff
@@ -59,6 +60,16 @@ def circum(self):
         """."""
         return _LSPEED * self.rev_time
 
+    @property
+    def sync_phase(self):
+        """Return the natural synchronous phase in rad.
+
+        Returns:
+            float: Natural synchronous phase [rad].
+
+        """
+        return _np.pi - _np.asin(self.en_lost_rad/self.gap_voltage)
```

Comment:
`numpy.arcsin()` is the arc sine function in numpy

Reply:
done!